glib/src/translate.rs

```diff
@@ -20,6 +20,72 @@
 //!     }
 //! ```
 //!
+//! Implementing `OptionToGlib` on a Rust type `T` allows specifying a sentinel to indicate
```

Comment:
Please use intra link (now that they're stable... :p). So for instance:

```
[`OptionToGlib`]
```

Reply:
Are you kidding me?! :) Last changes I'll do for this MR... After that I'll go on strike like a regular French guy.

Reply:
You can do it in a follow-up PR, I don't mind. But then don't forget to open an issue haha.